src/reproschema_to_fhir/fhir.py

```diff
@@ -159,7 +176,128 @@ def parse_reproschema_items(self, reproschema_content: OrderedDict):
             ...
         }
         """
-        raise NotImplementedError()
+        # there are a few possibilities for responses presented by reproschema:
+        # 1. responseOptions is a string, which is a reference to a file with the responses
+        # 2. responseOptions is a dict, which is a list of options
+
+        items = []
+
+        for item_path, item_json in reproschema_items.items():
+            curr_item = dict()
+
+            var_name = item_path.replace(f"items/", "")
+            curr_item[f"linkId"] = var_name
+
+            item_type = f"string"
```

Comment:
you can omit the `f` before the string if you are not including a variable. I think this is best practice. https://peps.python.org/pep-0498/